packages/trpc/server/routers/viewer/organizations/setPassword.handler.ts

```diff
@@ -0,0 +1,57 @@
+import { createHash } from "crypto";
+
+import { hashPassword } from "@calcom/features/auth/lib/hashPassword";
+import { verifyPassword } from "@calcom/features/auth/lib/verifyPassword";
+import { prisma } from "@calcom/prisma";
+
+import { TRPCError } from "@trpc/server";
+
+import type { TrpcSessionUser } from "../../../trpc";
+import type { TSetPasswordSchema } from "./setPassword.schema";
+
+type UpdateOptions = {
+  ctx: {
+    user: NonNullable<TrpcSessionUser>;
+  };
+  input: TSetPasswordSchema;
+};
+
+export const setPasswordHandler = async ({ ctx, input }: UpdateOptions) => {
```

Comment:
Sets the owner user password only used as part of the onboarding, which is why we check if the default password matched the one generated, otherwise the password can't be set by this method.